script.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable no-unused-vars */
+const addButton = document.getElementById('add-btn');
+const library = document.getElementById('library');
+
+let books = JSON.parse(localStorage.getItem('books'));
+
+function displayBooks() {
+  books.forEach((book) => {
+    library.insertAdjacentHTML('beforeend', `
+      <div>
+        <p>${book.Title}</p>
+        <p>${book.Author}</p>
+        <button onclick="removeBook('${book.Title}')">Remove</button>
+      </div>
+    `);
+  });
+}
+
+function updateLocalStorage() {
+  localStorage.setItem('books', JSON.stringify(books));
+}
+
+function reloadPage() {
+  document.location.reload(true);
+}
+
+function addBook(title, author) {
+  books.push({ Title: title, Author: author });
+  updateLocalStorage();
+  reloadPage();
+}
+
+function removeBook(title) {
+  books = books.filter((book) => book.Title !== title);
+  updateLocalStorage();
+  reloadPage();
+}
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you use a unique `id` property for each book since it would make it easier to remove books with duplicate values. For that you need to add a new property(`id`) in your `Book Object` then use it for your removing functionality instead of using `Title` property
```JavaScript
// Example
const addBook = (title, author) => {
  books.push({ id: Date.now(), Title: title, Author: author });
  ...
}

const removeBook = (id) => {
  books = books.filter((book) => book.id !== id);
 ...
}
```